share/functions/__fish_print_packages.fish

```diff
@@ -35,6 +35,17 @@ function __fish_print_packages
         return
     end
 
+    # pkg_info on OpenBSD provides versioning info which we want for
+    # installed packages
+    if begin
+            type -q -f pkg_add
+            and test (uname) = "OpenBSD"
```

Comment:
I like not using `uname` if we can somehow get away with it. Since we currently have no path for `pkg_add`, can we just remove this condition?

Reply:
IMO there needs to be some way to distinguish between pkg_* variants - for example pkg_add exists in pkgsrc and FreeBSD (this might be replaced with pkg by now.. not entirely sure.)

Reply:
Yeah, but we don't support those currently. We do support FreeBSD's `pkg`, so if they still have `pkg_add` and friends, we just need to make sure that `pkg` is checked first.

And if they have `pkg_*`, do they differ in ways that are meaningful here?